ckanext/canada/view.py

```diff
@@ -948,8 +949,15 @@ def _log_api_access(request_data, pkg_dict):
 
 
 def notice_no_access():
-    '''flash_notice if logged-in user can't actually do anything yet'''
-    if h.check_access('package_create'):
+    '''flash_notice if logged-in user can't actually do anything yet.'''
+    has_org_membership = model.Session.query(model.Member.id).filter(
```

Comment:
it's a bit of a layering violation to go straight to the db from a view. We could call the `organization_list_for_user` action instead so that this code doesn't break with future db changes

Reply:
Yes this action returns the list for the current user, not one passed in as a parameter. The performance shouldn't be an issue, it's only on login and register IIUC. If we wanted to add a parameter to return only the org ids from this action we could do that as a separate thing but it's not critical